src/components/templates/PosterPage/components/ShareModal/ShareModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC, useState, useCallback } from "react";
+import { Modal } from "react-bootstrap";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import { faFacebook, faTwitter } from "@fortawesome/free-brands-svg-icons";
+import { DEFAULT_MAP_BACKGROUND } from "settings";
+import { WithId } from "utils/id";
+import { PosterPageVenue } from "types/venues";
+import classNames from "classnames";
+import { LinkButton } from "components/atoms/LinkButton";
+
+import "./ShareModal.scss";
+
+interface ConfirmationModalProps {
+  show: boolean;
+  onHide: () => void;
+  venue: WithId<PosterPageVenue>;
+}
+
+export const ShareModal: FC<ConfirmationModalProps> = ({
+  show,
+  onHide,
+  venue,
+}) => {
+  const url = `${process.env.REACT_APP_CODE_CHECK_URL}in/${venue.id}`;
+  const [isCopied, setIsCopied] = useState(false);
+
+  const handlerCopied = useCallback(() => {
+    setIsCopied(true);
+  }, []);
+
+  const linkText = isCopied ? "Link Copied" : "Copy Link";
+
+  const linkClasses = classNames("share-modal-container__link", {
+    "share-modal-container__link--copied": isCopied,
+  });
+
+  const textTitle = `Check out this OHBM Poster, ${venue.poster?.title} by ${venue.poster?.authorName} at ${url}`;
+  const facebookHref = `https://www.facebook.com/sharer/sharer.php?app_id=${process.env.REACT_APP_FACEBOOK_APP_ID}&u=${url}&quote=${textTitle}&thumbnail=${venue.host?.icon}&picture=${venue.host?.icon}&hashtag=#OHBM2021`;
+  const twitterHref = `https://twitter.com/intent/tweet?text=${textTitle}&hashtags=OHBM2021`;
+
+  return (
+    <Modal show={show} onHide={onHide}>
+      <Modal.Body>
+        <div className="share-modal-header">
```

Comment:
ShareModal__header and other classnames

Reply:
Following our BEM naming standards:

- http://getbem.com/naming/

Reply:
https://github.com/sparkletown/sparkle/pull/1396/commits/8c232a5d1ae872885bf6d3d3e41c8104f633279b